CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing to api-designer
+
+Do you want to help improving this project?? Here are some instructions to get you started. They are probably not perfect, please let us know if anything feels wrong or incomplete.
+
+## Reporting Issues
+
+When reporting [issues](https://github.com/mulesoft/api-designer/issues)
+on GitHub please include the version of api-designer you are using, as well as the name and version of your browser, and your OS. Please also include the steps required to reproduce the problem if possible and applicable. This information will help us review and fix your issue faster.
+
+## Build Environment
+
+Install global tools
+
+```
```

Comment:
No worries. This one I noticed because of the comment (handy to syntax highlight it), but it's applicable to any of the shell scripts.